LeagueToolkit/Helpers/Structures/Color.cs

```diff
@@ -202,6 +200,19 @@ public bool Equals(Color other)
                 this.A == other.A;
         }
 
+        public override bool Equals(object obj)
+        {
+            if (obj is null) return false;
+            if (ReferenceEquals(this, obj)) return true;
+            if (obj.GetType() != this.GetType()) return false;
+            return Equals((Color)obj);
+        }
```

Comment:
Rider suggests
```cs
        public override bool Equals(object obj)
        {
            return obj is Color other && Equals(other);
        }
```
for me, which is equally correct but much simpler. Besides, at least the `if (ReferenceEquals(this, obj))` is always false.

Reply:
have changed it accordingly
(I myself do not use Rider, so I did not have the suggestion)